gdx/res/com/badlogic/gdx.gwt.xml

```diff
@@ -354,6 +354,7 @@
 		<include name="math/DelaunayTriangulator.java"/>
 		<include name="math/EarClippingTriangulator.java"/>
 		<include name="math/Ellipse.java"/>
+		<include name="math/FastNoiseLite.java"/>
```

Comment:
Good catch on the .gwt.xml file! This is an easy file to miss when updating. This is good!